### PR TITLE
[BUGFIX] Get `MicrosoftTeamsNotificationAction` working with V1

### PR DESCRIPTION
**Pull request by cdkini**
Overhaul to improve UI but also fix to get things functional in V1

Cloud
Screenshot 2024-10-29 at 4 00 45 PM
Core
Screenshot 2024-10-29 at 4 02 28 PM

- [x] Description of PR changes above includes a link to [an existing GitHub issue](https://github.com/great-expectations/great_expectations/issues)
- [x] PR title is prefixed with one of: [BUGFIX], [FEATURE], [DOCS], [MAINTENANCE], [CONTRIB]
- [x] Code is linted - run `invoke lint` (uses `ruff format` + `ruff check`)
- [x] Appropriate tests and docs have been updated

For more information about contributing, see [Contribute](https://docs.greatexpectations.io/docs/contributing/contributing_checklist).

After you submit your PR, keep the page open and **monitor the statuses of the various checks made by our continuous integration process at the bottom of the page. Please fix any issues that come up** and [reach out on Slack](https://greatexpectations.io/slack) if you need help. Thanks for contributing!

### CODE REVIEW

File: great_expectations/render/renderer/microsoft_teams_renderer.py [hunk 1]
```diff
@@ -20,196 +20,190 @@
...
+
+    Relevant links/documentation:
+        * Payload schema: https://adaptivecards.io/explorer/
+        * Interactive UI editor: https://adaptivecards.io/designer/
+
+    """
+
+    _MICROSOFT_TEAMS_CONTENT_TYPE = "application/vnd.microsoft.card.adaptive"
+    _MICROSOFT_TEAMS_SCHEMA_URL = "http://adaptivecards.io/schemas/adaptive-card.json"
+    _MICROSOFT_TEAMS_SCHEMA_VERSION = 1.5
```
Comment (cdkini) [inline great_expectations/render/renderer/microsoft_teams_renderer.py:35]: 1.6 is latest in the MS ecosystem but 1.5 is the latest that Teams supports

File: great_expectations/render/renderer/microsoft_teams_renderer.py [hunk 2]
```diff
@@ -20,196 +20,190 @@
...
+    _GX_LOGO_URL = "https://pbs.twimg.com/profile_images/1600908333493338112/GQQEPSKg_400x400.jpg"
```
Comment (cdkini) [inline great_expectations/render/renderer/microsoft_teams_renderer.py]: Picked a random GX image from google images but hoping to find a more official URL

Comment (tyler-hoffman) [reply great_expectations/render/renderer/microsoft_teams_renderer.py]: I think we should pick an image we host.

**Review by tyler-hoffman — APPROVED** [review]
Approved pending the url gets updated

**Pull request merged**